### ISSUE TITLE
Notify if background sync updates the local policy

### ISSUE THREAD

**Issue by olensmar** [kubeshop/vscode-monokle#26]
As a dev I would like to know when/if the policy for my project is changed during synchronization - suggest to show some kind of notification - "The policy for your repo has been updated automatically from Monokle Cloud".. 

could we show exactly what has changed? separate issue?

**Comment by f1ames** [comment]
> could we show exactly what has changed? separate issue?

Yes, we can diff local policy with what was fetched. Probably it could be part of `@monokle/synchronizer` (just diffing) as it already emits `synchronize` event which could additionally include such data. We can extract separate task when starting work on this one.

Also it could be a good idea to keep update datetime (or the time remote policy was last checked for changes) and show it somehow to users.
